### PR TITLE
Remove duplicate functions

### PR DESCRIPTION
**Pull request by tupaschoal**
Just some small changes that extract equal code to functions or calls another one instead of doing exactly the same thing.

### CODE REVIEW

File: src/openrct2/network/Twitch.cpp [hunk 1]
```diff
@@ -286,12 +283,12 @@ namespace Twitch
...
             return;
         }
-        snprintf(url, sizeof(url), "%s/channel/%s/audience", gConfigTwitch.api_url, gConfigTwitch.channel);
+        snprintf(url, sizeof(url), requestFormat.c_str(), gConfigTwitch.api_url, gConfigTwitch.channel);
 
         _twitchState = TWITCH_STATE_WAITING;
         _twitchIdle = false;
 
-        Http::DoAsync({ url }, [](Http::Response res) {
+        Http::DoAsync({ url }, [&](Http::Response res) {
```
Comment (janisozaur) [inline src/openrct2/network/Twitch.cpp]: This is for actually modifying `_twitchState`, `_twitchJsonResponse` I presume?

Comment (tupaschoal) [reply src/openrct2/network/Twitch.cpp]: Kind of, is for the lambda to have access to `operation` to use it on the assignment

Comment (janisozaur) [reply src/openrct2/network/Twitch.cpp]: It would have had access to `operation` with value capture anyway.

Comment (tupaschoal) [reply src/openrct2/network/Twitch.cpp]: Indeed, didn't give too much thought, will change it

**Comment by tupaschoal** [comment]
Is this ok for merging now @janisozaur ?

**Comment by janisozaur** [comment]
I think so, as long as Travis is green

**Comment by tupaschoal** [comment]
Ok!

File: src/openrct2/world/Map.h [hunk 2]
```diff
@@ -176,6 +176,7 @@ TileElement* tile_element_insert(const TileCoordsXYZ& loc, int32_t occupiedQuadr
 
 using CLEAR_FUNC = int32_t (*)(TileElement** tile_element, int32_t x, int32_t y, uint8_t flags, money32* price);
 
+int32_t map_place_clear_func(TileElement** tile_element, int32_t x, int32_t y, uint8_t flags, money32* price, bool is_scenery);
```
Comment (duncanspumpkin) [inline src/openrct2/world/Map.h]: Why was this added to the header? It will never be used.

Comment (tupaschoal) [reply src/openrct2/world/Map.h]: You're correct, it was not needed, I removed it.

**Comment by tupaschoal** [comment]
I guess it does need the function on the header (which I suspected it would - see all the failures on Travis). So I added it back again

**Comment by duncanspumpkin** [comment]
No that isn't right. Its a static function. You need to declare it in the file.  Just add `static`.
